### PR TITLE
feat: add new functions to cel-go docs

### PR DESCRIPTION
**Pull request by adityathebe**
resolves: https://github.com/flanksource/docs/issues/106

### CODE REVIEW

File: mission-control/docs/reference/scripting/cel.md [hunk 1]
```diff
@@ -19,7 +18,108 @@ spec:
 ```
 
 ---
-## Collections
```
Comment (moshloop) [inline mission-control/docs/reference/scripting/cel.md:22]: This should go into canary-checker, the Makefile copies the canary scripting file to mission control docs

File: mission-control/docs/reference/scripting/cel.md [hunk 2]
```diff
@@ -19,7 +18,108 @@ spec:
...
+  "region": string,
+  "account": string,
+  "resource": string,
+}
+```
+
+Examples:
+
+```javascript
+arnToMap('arn:aws:sns:eu-west-1:123:MMS-Topic').region == 'eu-west-1'; // evaluates to true
```
Comment (moshloop) [inline mission-control/docs/reference/scripting/cel.md:43]:
```suggestion
arnToMap('arn:aws:sns:eu-west-1:123:MMS-Topic').region  // eu-west-1
```

File: canary-checker/docs/scripting/cel.md [hunk 3]
```diff
@@ -1478,1840 +867,1991 @@ Parsing with hour and minute for a specific zone:
...
-- `futureTime` is a `time.Time` object representing a future point in time.
+- `jsonString` is the JSON formatted string.
 
 Examples:
 
 ```javascript
-Calculating the time remaining until a specific future time:
-time.Until(time.Parse("2006-01-02", "2023-10-01"))  // Evaluates to the duration until October 1, 2023
+Converting a JSON string to a map:
+JSON("{\"name\": \"Alice\", \"age\": 30}")  // Evaluates to a map with keys "name" and "age"
```
Comment (moshloop) [inline canary-checker/docs/scripting/cel.md:1203]: Can we use the piped version by default .e.g 

`"{\"name\": \"Alice\", \"age\": 30}".JSON() `

and then

use the comment to display the exact result, which negates the need for a comment most of the time, if a comment is needed before, then it should be a comment and not just text.  i.e. the expression should be copy and pasteable

File: canary-checker/docs/scripting/cel.md [hunk 4]
```diff
@@ -1478,1840 +867,1991 @@ Parsing with hour and minute for a specific zone:
...
 
-Creating a 10-character random alphabetic string:
-random.Alpha(10)  // Might evaluate to "FgHiJkLmNo"
+Using tab for indentation:
+toJSONPretty("\t", ["Alice", 30])  // Evaluates to a pretty-formatted JSON string with tabs
 
-Producing a 15-character random alphabetic string:
-random.Alpha(15)  // Might evaluate to "pQrStUvWxYzAbCdEf"
+An empty map with four spaces indent:
+toJSONPretty("    ", {})  // Evaluates to "{}" with no indents as the map is empty
```
Comment (moshloop) [inline canary-checker/docs/scripting/cel.md:1263]: I think we are missing toJSON()

**Comment by adityathebe** [comment]
@moshloop Should we remove functions that are already methods?

Eg: `toJSON`, `JSON`, `JSONArray`.

I think we could also remove `humanDuration/humanSize` since we already have `HumanDuration/HumanSize`. From what I am seeing, we can go with the convention of
- methods -> camel case
- functions -> Pascal case

Also we could namespace the aws and k8s functions. `k8s.GetStatus` instead of `GetStatus`

File: canary-checker/docs/scripting/cel.md (hunk 3 above)
Comment (adityathebe) [reply canary-checker/docs/scripting/cel.md]: https://github.com/flanksource/gomplate/pull/37

**Pull request merged**